wiki/Tournaments/SS_Tournament/2018/en.md

```diff
@@ -40,7 +71,7 @@ Brackets will then be made with the top placing team in group A vs the second to
 * Quarter Finals and Semi Finals will be best of 9
 * Finals will be best of 11
 * Grand Finals will be best of 13
-* In the case of a tie in group stage, team advancement will be based off a team's '[map wins - map losses = Points]' where higher points advance. If the score is equal, the score of Team A Vs Team B will be taken into account, i.e. Team A and Team B both have 6 wins and 6 losses, but Team A beat Team B 3-2 Team A will advance. If there is still a tie it will be average score difference gathered by the formula '[Team A's scores of all played maps - All opposing teams scores / maps played]'. If there is still a tie after this, it will be based off seeding.
+* In the case of a tie in group stage, team advancement will be based off a team's '[map wins - map losses = Points]' where higher points advance. If the score is equal, the score of Team A Vs Team B will be taken into account, i.e. Team A and Team B both have 6 wins and 6 losses, but Team A beat Team B 3-2 Team A will advance.If there is still a tie it will be average score difference gathered by the formula '[Team A's scores of all played maps - All opposing teams scores / maps played]'.If there is still a tie after this, it will be based off seeding.
```

Comment:
super minor but heh - add a space before "If there is still a tie after this, it will be based off seeding."